dist/morphext.js

```diff
@@ -22,6 +22,8 @@
             animation: "bounceIn",
             separator: ",",
             speed: 2000,
+            activeClass: 'active',
```

Comment:
Can we use double quotes here for consistency? I think with the new ESLint configuration, this should be caught.